src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -63,7 +63,11 @@
 					v-show="needsNews"
 					v-model="newsConsent"
 				>
-					I want to receive updates about my loans, Kiva news, and promotions in my inbox
+					{{ !passwordless
+						? 'I want to receive updates about my loans, Kiva news, and promotions in my inbox'
+						: `Receive email updates from Kiva (including borrower updates and promos).
+							You can unsubscribe anytime. (optional)`
```

Comment:
Again, I think we need to keep the `You can unsubscribe anytime. (optional)` portion of the this copy.

Reply:
Oh wait, this is a new addition. Sorry.

Reply:
We probably should add the `You can unsubscribe anytime. (optional)` portion to the existing copy as well.

Reply:
won't do since comment sounds like it was made in error

Reply:
I suspect the additional copy is coming from Legal recommendations which is why I think we might want to always show for all users.